### ISSUE TITLE
Support Kotlin 1.2 (Not a valid Kotlin file error)

### ISSUE THREAD

**Issue by deviant-studio** [shyiko/ktlint#89]
```
@Module(includes = [LoginInteractorModule::class])
abstract class LoginModule {

    @Binds
    abstract fun view(view: LoginActivity): LoginView

}
```

LoginModule.kt:8:19: Not a valid Kotlin file (expecting an expression)

**Comment by shyiko** [comment]
Hi @deviant-studio. kotlin@1.2 will be supported as soon as it's available in Maven Central (in other words, as soon as it's out of EAP).

**Comment by vRallev** [comment]
The RC has been released. Does this make a difference?

**Comment by shyiko** [comment]
@vRallev unfortunately no. It's still not available it Maven Central. The reason why we wait for stable release(s) is to avoid https://github.com/shyiko/ktlint/issues/31.

**Comment by shyiko** [comment]
Included in [ktlint@0.13.0](https://github.com/shyiko/ktlint/releases/tag/0.13.0) :tada:
